test/suites/reboot.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Documentation    Tests related to system reboot
+
+Resource            ../resources/common.resource
+Resource            ../resources/oc.resource
+Resource            ../resources/ostree-health.resource
+
+
+*** Variables ***
+${SOURCE_POD}        ./assets/reboot/pod-with-pvc.yaml
+${POD_NAME_STATIC}    test-pod
+${NAMESPACE}          default
```

Comment:
Would it be better to create a dedicated namespace for this test? 
We've got `Create Random Namespace` function in `kubeconfig.resource.

Reply:
i can use `Suite Setup         Setup Suite With Namespace` at the beginning ..